
A Promscale test has a dependency on a package which was not intended to be
vendored, so `go mod vendor` breaks. The dependency is not required to build
Promscale, so it can be removed.

diff --git a/pkg/tests/end_to_end_tests/trace_query_integration_test.go b/pkg/tests/end_to_end_tests/trace_query_integration_test.go
index 84a0d08..efbe5e1 100644
--- a/pkg/tests/end_to_end_tests/trace_query_integration_test.go
+++ b/pkg/tests/end_to_end_tests/trace_query_integration_test.go
@@ -24,7 +24,6 @@ import (
 	ingstr "github.com/timescale/promscale/pkg/pgmodel/ingestor"
 	"github.com/timescale/promscale/pkg/pgxconn"
 
-	jaegerQueryApp "github.com/jaegertracing/jaeger/cmd/query/app"
 	jaegerQueryService "github.com/jaegertracing/jaeger/cmd/query/app/querysvc"
 	jaegerJSONModel "github.com/jaegertracing/jaeger/model/json"
 )
